### PR TITLE
Drop support for init scripts for Foreman

### PR DESCRIPTION
**Pull request by ehelms**
@evgeni Follow up for init scripts

### CODE REVIEW

**Review by tbrisker — APPROVED** [review]
I'm fine with dropping them, please just add an upgrade warning in the manual.

File: debian/bionic/foreman/foreman.preinst [hunk 1]
```diff
@@ -16,6 +16,8 @@ fi
 update-rc.d dynflowd remove >/dev/null
 dpkg-maintscript-helper rm_conffile /etc/init.d/dynflowd 2.0.0~ -- "$@"
 dpkg-maintscript-helper rm_conffile /etc/default/dynflowd 2.0.0~ -- "$@"
+dpkg-maintscript-helper rm_conffile /etc/init.d/foreman 2.3.0~ -- "$@"
+dpkg-maintscript-helper rm_conffile /etc/default/foreman 2.3.0~ -- "$@"
```
Comment (ekohl) [inline debian/bionic/foreman/foreman.preinst:20]: Now that we've branched, should this be `2.4.0`? @evgeni?

**Comment by ekohl** [comment]
I opened #6045 which includes this PR (updated with the version number) and a Redmine issue.

File: debian/bionic/foreman/foreman.preinst (hunk 1 above)
Comment (evgeni) [reply debian/bionic/foreman/foreman.preinst:20]: Yes

**Comment by evgeni** [comment]
closing here then

**Pull request closed without merging**